### PR TITLE
Feature/update media

### PR DESCRIPTION
**Pull request by eevargas**
Hey Peter, I think I addressed all the comments on your code. I added some comments, but just look that the code and let me know if you have any questions.

### CODE REVIEW

**Review by peterztan — APPROVED** [review]
Thank you so much Eli, if I have time I'll most likely keep making changes (maybe finally get the audio and image dynamic path working, if you have time would you be able to look them over for me?

**Comment by eevargas** [comment]
I did, they are working on this PR.

File: assets/javascript/game.js [hunk 1]
```diff
@@ -27,111 +27,134 @@ var triedLetters = [];
...
 var wins = 0;
-var placeHolder = " " + "_" + " ";
+var losses = 0;
+var placeHolder = " _ "; // no need to concatenate here
+
+var songImage = document.getElementById('songImage')
+var audio = document.getElementById('currentSong')
+audio.volume = 0.25;
+
+function setMedia (idx) {
```
Comment (eevargas) [inline assets/javascript/game.js:46]: Image and audio, dynamic load.

**Review by peterztan — APPROVED** [review]

File: assets/javascript/game.js (hunk 1 above)
Comment (peterztan) [reply assets/javascript/game.js:46]: Thank you Eli! :)

**Pull request merged**